### ISSUE TITLE
batchNormalization has different parameter order than tensorflow

### ISSUE THREAD

**Issue by kylemcdonald** [tensorflow/tfjs#314]
Using tfjs-0.11.0, [tf.batchNormalization](https://js.tensorflow.org/api/0.11.1/#batchNormalization) has the parameters: `tf.batchNormalization (x, mean, variance, varianceEpsilon?, scale?, offset?)`

[tf.nn.batch_normalization](https://www.tensorflow.org/api_docs/python/tf/nn/batch_normalization) has the parameters: `tf.nn.batch_normalization(x, mean, variance, offset, scale, variance_epsilon)`.

variance_epsilon is in a different spot, but more importantly: scale and offset are reversed. This tripped me up for a while when I was trying to port code. This is a backwards incompatible change, so maybe it's better to just add a note to the documentation pointing out that this order is different?

**Comment by bileschi** [comment]
Thanks Kyle!  Will take a look

**Comment by ManrajGrover** [comment]
I feel we should fix this and add note for breaking change in release notes.

**Comment by nsthorat** [comment]
Yeah, this is my fault from a long time ago before we were called TensorFlow. This is a super tricky one because we can swap the order, note a breaking change, but it will silently succeed for people depending on this ordering right now.

I'm thinking that this might be a version 1.0 thing since we can break the API and the converter tools / layers will be more mature, so less likely people will use the low-level op directly. For now, we should prioritize adding some big documentation to batchNorm.
